packages/react-union/API.md

```diff
@@ -197,13 +197,13 @@ If you need to provide custom data to the instance of your widget, pass a valid
 </script>
 ```
 
-**Data Widget**
+**Passing common data to all widgets**
 
-If you only need to provide data to other widgets in your application, there is no need to render anything.
+If you need to provide common data to all widgets in your application, use the `data-union-common` attribute. This data will be recursively merged with the JSON content of each widget descriptor.
```

Comment:
Please, add info, that is encouraged to use just one common config at the view if the rendering engine cannot ensure the order of rendering.

Reply:
Good point! Will do.